README.md

```diff
@@ -1,133 +1,61 @@
+- Wrote capybara test. RED: Test failed
+- Added welcome return to ‘/login’ route in app.rb
+- Feature test passed. GREEN: Test passed
+<hr>
 
-Bonus:
------
+- Creating a Chitter database
+- Database called chitter created in Postgresql.
+- Extracting Logic to the Model. (rewrite bullet points)
+- Wrote a unit test for Account instance to return username. RED. Test failed.
+- Defined Account class in lib folder with correct return. GREEN. Test passed.
 
-If you have time you can implement the following:
+### User story 2
 
-* In order to start a conversation as a maker I want to reply to a peep from another maker.
-
-And/Or:
-
-* Work on the CSS to make it look good.
-
-Good luck and let the chitter begin!
-
-Code Review
------------
-
-In code review we'll be hoping to see:
-
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/master/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
-
-Reviewers will potentially be using this [code review rubric](docs/review.md).  Referring to this rubric in advance may make the challenge somewhat easier.  You should be the judge of how much challenge you want this weekend.
-
-Automated Tests:
------
-
-Opening a pull request against this repository will will trigger Travis CI to perform a build of your application and run your full suite of RSpec tests. If any of your tests rely on a connection with your database - and they should - this is likely to cause a problem. The build of your application created by has no connection to the local database you will have created on your machine, so when your tests try to interact with it they'll be unable to do so and will fail.
-
-If you want a green tick against your pull request you'll need to configure Travis' build process by adding the necessary steps for creating your database to the `.travis.yml` file.
-
-- [Travis Basics](https://docs.travis-ci.com/user/tutorial/)
-- [Travis - Setting up Databases](https://docs.travis-ci.com/user/database-setup/)
-
-Notes on test coverage
-----------------------
-
-Please ensure you have the following **AT THE TOP** of your spec_helper.rb in order to have test coverage stats generated
-on your pull request:
-
-```ruby
-require 'simplecov'
-require 'simplecov-console'
-
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new([
-  SimpleCov::Formatter::Console,
-  # Want a nice code coverage website? Uncomment this next line!
-  # SimpleCov::Formatter::HTMLFormatter
-])
-SimpleCov.start
+```
+As a Maker
+So that I can let people know what I am doing  
+I want to post a message (peep) to chitter
 ```
 
-You can see your test coverage when you run your tests. If you want this in a graphical form, uncomment the `HTMLFormatter` line and see what happens!
+- Feature test: Maker can post a peep. 
+- Wrote feature test: User can submit message to send. RED: Test failed.
```

Comment:
Overall amazing README, easy to follow and understand your process and way of solving this challenge 👍